infer.sh

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env bash
 ./gradlew clean && infer --disable-issue-type "NULL_DEREFERENCE" \
---disable-issue-type "RESOURCE_LEAK" -- ./gradlew sdk:build
+--disable-issue-type "RESOURCE_LEAK" \
+--threadsafe-aliases "[\"ThreadSafe\"]" \
```

Comment:
adds `com.bugsnag.android.ThreadSafe` as an alias for `com.facebook.infer.annotation.ThreadSafe`